api/v1alpha1/internalservicesconfig_types.go

```diff
@@ -25,7 +25,7 @@ const InternalServicesConfigResourceName string = "config"
 // InternalServicesConfigSpec defines the desired state of InternalServicesConfig.
 type InternalServicesConfigSpec struct {
 	// AllowList is the list of remote namespaces that are allowed to execute InternalRequests
-	// +required
+	// +optional
```

Comment:
Is that correct, changing it to optional? If yes, maybe it should be in a different PR.